### PR TITLE
Issue 112

### LINKED ISSUE #397: [112] Issue #112 - June 28, 2018
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by BasThomas**
Closes #397

### CODE REVIEW

File: _drafts/2018-06-28-issue-112.md [hunk 1]
```diff
@@ -4,47 +4,66 @@ title: ! 'Issue #112'
 author: btb
 ---
 
-> TODO: intro comments
+I am back! I've had the most wonderful time at WWDC, learning a lot of and meeting awesome people.
```
Comment (jessesquires) [inline _drafts/2018-06-28-issue-112.md]: I've had --> I had

File: _drafts/2018-06-28-issue-112.md [hunk 2]
```diff
@@ -4,47 +4,66 @@ title: ! 'Issue #112'
...
+I want to specifically thank all those that have shown their gratitude for the weekly brief: it's much appreciated!
```
Comment (jessesquires) [inline _drafts/2018-06-28-issue-112.md]: I'd replace the `:` with a `,` (or period)

File: _drafts/2018-06-28-issue-112.md [hunk 3]
```diff
@@ -4,47 +4,66 @@ title: ! 'Issue #112'
...
+- [SR-8049](https://bugs.swift.org/browse/SR-8049) [Swift for TensorFlow] Open source S4TF unit testing with a remote TF session
 
 {% include task_remind.html %}
 
 ### Swift Unwrapped
 
-> TODO: Latest episode(s) of Swift Unwrapped
+In [episode 63](https://spec.fm/podcasts/swift-unwrapped/147483), Jesse & JP discuss Swift algorithms and data structures with [Kelvin Lau](https://twitter.com/KelvinlauKl) & [Vincent Ngo](https://twitter.com/VincentNgo2).
+
+In [episode 64](https://spec.fm/podcasts/swift-unwrapped/160151), Jesse & JP discuss `Never`, the [return type of functions that do not return normally](https://developer.apple.com/documentation/swift/never).
```
Comment (jessesquires) [inline _drafts/2018-06-28-issue-112.md]: spell-out "and" instead of using `&`

File: _drafts/2018-06-28-issue-112.md [hunk 4]
```diff
@@ -4,47 +4,66 @@ title: ! 'Issue #112'
...
 
-> TODO
+[Ole Begemann](https://twitter.com/olebegemann) wrote [a blog post](https://oleb.net/blog/2018/06/dynamic-member-lookup/) with thoughts on `@dynamicMemberLookup`, which was introduced in [SE-0195](https://github.com/apple/swift-evolution/blob/master/proposals/0195-dynamic-member-lookup.md).
+
+[John Sundell](https://twitter.com/johnsundell) discussed the Swift Package Manager with [David Hart](https://twitter.com/dhartbit) in his [podcast](https://www.swiftbysundell.com/podcast/26).
 
 ### Commits and pull requests
 
-> TODO
+[Slava Pestov](https://github.com/slavapestov) merged [a pull request](https://github.com/apple/swift/pull/17227) to fix a corner case for source compatibility. As it appears, using an enum case called `init` could cause some trouble.
```
Comment (jessesquires) [inline _drafts/2018-06-28-issue-112.md]: lol. amazing

File: _drafts/2018-06-28-issue-112.md [hunk 5]
```diff
@@ -4,47 +4,66 @@ title: ! 'Issue #112'
...
 
-> TODO
+> - Improved scheme generation logic
+> - Local dependencies
+> - Automatic Xcode project generation
+> - System library targets
+> - Embeddable Xcode projects
 
-### Swift Forums
+[Dave DeLong](https://twitter.com/davedelong) & [Erica Sadun](https://twitter.com/ericasadun) pitched [a proposal](https://forums.swift.org/t/introducing-namespacing-for-common-swift-error-scenarios/10773) to introduce namespacing for common Swift error scenarios.
```
Comment (jessesquires) [inline _drafts/2018-06-28-issue-112.md]: another `&`

File: _drafts/2018-06-28-issue-112.md (hunk 4 above)
Comment (BasThomas) [reply _drafts/2018-06-28-issue-112.md]: Right? 🤣

**Comment by BasThomas** [comment]
Thanks for the review! :)

**Pull request merged**